client/orm/models.go

```diff
@@ -353,6 +353,9 @@ func (mc *_modelCache) register(prefixOrSuffixStr string, prefixOrSuffix bool, m
 			return
 		}
 
+		if val.Elem().Kind() == reflect.Slice {
```

Comment:
I can not understand this part. Do you mean that model could be Array or Slice?

Reply:
Yes, for example
```go
var sun []Sum
orm.Aggregate("dept_name,sum(salary) as total").GroupBy("dept_name").OrderBy("dept_name").All(&sun)
```
`[]Sum` is a dynamic registration model, so add this judgment.

Reply:
How about 
```
var sun []Sum
Register(&Sum{})
orm.Aggregate("dept_name,sum(salary) as total").GroupBy("dept_name").OrderBy("dept_name").All(&sun)
```
Because I think registering a slice looks not good.

Reply:
No, this logic is to extract the struct information.
```go
if val.Elem().Kind() == reflect.Slice {
   val = reflect.New(val.Elem().Type().Elem())
}
```